[2022-09-14T13:15:51.213+0300] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: First-Dag.RandomNumberGeneratorB scheduled__2022-09-14T10:10:00+00:00 [queued]>
[2022-09-14T13:15:51.221+0300] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: First-Dag.RandomNumberGeneratorB scheduled__2022-09-14T10:10:00+00:00 [queued]>
[2022-09-14T13:15:51.221+0300] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-14T13:15:51.221+0300] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-14T13:15:51.221+0300] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-14T13:15:51.242+0300] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): RandomNumberGeneratorB> on 2022-09-14 10:10:00+00:00
[2022-09-14T13:15:51.243+0300] {standard_task_runner.py:52} INFO - Started process 13625 to run task
[2022-09-14T13:15:51.245+0300] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'First-Dag', 'RandomNumberGeneratorB', 'scheduled__2022-09-14T10:10:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/First_Dag.py', '--cfg-path', '/tmp/tmpqutt0zr8', '--error-file', '/tmp/tmpht_zti61']
[2022-09-14T13:15:51.246+0300] {standard_task_runner.py:80} INFO - Job 75: Subtask RandomNumberGeneratorB
[2022-09-14T13:15:51.412+0300] {task_command.py:371} INFO - Running <TaskInstance: First-Dag.RandomNumberGeneratorB scheduled__2022-09-14T10:10:00+00:00 [running]> on host DESKTOP-6F0IB4Q.localdomain
[2022-09-14T13:15:51.467+0300] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=First-Dag
AIRFLOW_CTX_TASK_ID=RandomNumberGeneratorB
AIRFLOW_CTX_EXECUTION_DATE=2022-09-14T10:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-14T10:10:00+00:00
[2022-09-14T13:15:51.468+0300] {logging_mixin.py:115} INFO - Number B is: 0
[2022-09-14T13:15:51.468+0300] {python.py:173} INFO - Done. Returned value was: 0
[2022-09-14T13:15:51.487+0300] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=First-Dag, task_id=RandomNumberGeneratorB, execution_date=20220914T101000, start_date=20220914T101551, end_date=20220914T101551
[2022-09-14T13:15:51.538+0300] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-14T13:15:51.556+0300] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
